synapse/lib/ast.py

```diff
@@ -1900,7 +1900,8 @@ async def run(self, runt, genr):
                 yield node, path
 
             if not isinstance(node.form.type, s_types.Edge):
-                continue
+                mesg = f'Pivot in to a specific form cannot be used with nodes of type {node.form.type.name}'
```

Comment:
Would this be `pivot in from` ? Maybe the analysts have a precedent they've used in docs?

Reply:
updated per discussion